cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -949,7 +949,7 @@ pTxIn =
 
 pTxOut :: Parser TxOut
 pTxOut =
-  Opt.option (Opt.maybeReader (parseTxOut . Text.pack))
+  Opt.option (Opt.eitherReader (\str -> parseTxOut $ Text.pack str))
```

Comment:
You should be able to do `Opt.option (Opt.eitherReader (parseTxOut . Text.pack))`